.github/workflows/production.yml

```diff
@@ -11,7 +11,7 @@ env:
   AZURE_RESOURCE_GROUP: rg-theme-prod-eastus
   NODE_ENV: production
   NODE_VERSION: 13.x
-  SOURCE_DIRECTORY: guide
+  SOURCE_DIRECTORY: fractal-build
   STORAGE_ACCOUNT: stgthemeprodeastus
   STORAGE_CONTAINER: \$web
 
```

Comment:
The GH pages step is unnecessary if this is to take the place of the previous "guide" entirely. I'd also remove the guide directory as well.

Reply:
What about for branch previews?

Reply:
We were only using GH pages for the production build. The branch previews are Azure.